docs/conf.py

```diff
@@ -68,7 +68,13 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = "furo"
```

Comment:
I think I prefer the RTD theme but I don't feel strongly about this...

@norabelrose @levmckinney @yawen-d any preference? You can see examples at https://imitation--484.org.readthedocs.build/en/484/getting-started/first-steps.html vs https://imitation.readthedocs.io/en/latest/guide/gettingstarted.html

Reply:
My intention was to make the docs look a bit cleaner, more elegant/modern, as well as appealing to read. Given that they currently function as our project 'website', I think that making it welcoming is a goal to keep in mind. Furo is very-well supported and used by a bunch of big projects (pip, black, python dev guide, psycopg...) But If furo is not much of your liking, there's also the [Pydata theme](https://pydata-sphinx-theme.readthedocs.io/) (used by numpy, pandas, scipy, matplotlib...)

It is true that RTD theme has this 'the original theme' vibe, and there's something to be said about that. But I find RTD theme to have many UX issues: there's no dark mode, the body is not centered (so hard to read in large monitors), it squeezes the TOC of the current page within the TOC of the site (instead of as a right sidebar), the font of the sidebar on multiple levels of indent becomes hard to read, it doesn't track the section you're watching based on scroll position...

Reply:
`furo` has grown on me a bit now that I look at it again, so I think I'm OK with the change.